app/views/layouts/application.html.erb

```diff
@@ -4,17 +4,15 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>Fabricator</title>
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css" integrity="sha384-WskhaSGFgHYWDcbwN70/dfYBj47jz9qbsMId/iRN3ewGhXQFZCSftd1LZCfmhktB" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css"
```

Comment:
Can we get rid of this now that we're importing Bootstrap through Reactstrap?

Reply:
Yes, I'll remove it. I also should check once that the bootstrap styles still get imported correctly, so that everything looks normal in the app as well, not just in storybook.

Reply:
@angelocordon actually, I can't get the direct bootstrap import working for some reason. How about we just keep the cdn instead for now? Won't hurt